kolibri/content/models.py

```diff
@@ -170,11 +177,14 @@ def save(self, *args, **kwargs):
             md5 = hashlib.md5()
             for chunk in self.content_copy.chunks():
                 md5.update(chunk)
-
             self.checksum = md5.hexdigest()
             self.available = True
             self.file_size = self.content_copy.size
-            self.extension = os.path.splitext(self.content_copy.name)[1]
+            # if feed a format that doesn't exist yet will break the app
+            # not sure how we want to handle this yet
+            # import pdb
+            # pdb.set_trace()
+            self.file_format = FileFormat.objects.using(channel_id).get(extension=self.content_copy.name.split('.')[1])
```

Comment:
Should use [`splitext`](https://docs.python.org/2/library/os.path.html#os.path.splitext) (like in the old code). This code will break if there's a dot earlier in the filename.